addons/common/XEH_postInit.sqf

```diff
@@ -27,6 +27,17 @@
     GVAR(nextFrameBufferA) = GVAR(nextFrameBufferB);
     GVAR(nextFrameBufferB) = [];
     GVAR(nextFrameNo) = diag_frameno + 1;
+
+    //Handle the waitUntilAndExec array:
+    _deleted = 0;
+    {
+        _x params ["_condition", "_code", "_args"];
+        if ((_args call _condition)) then {
+            GVAR(waitUntilAndExecArray) deleteAt (_forEachIndex - _deleted);
+            _deleted = _deleted + 1;
+            _args call _code;
```

Comment:
"code" is a data type and therefore a bad variable name. The condition is code too. We usually use `condition` and `statement` for a construct like this.
Also consider not saving these in variables, because
`if ((_this select 2) call (_his select 0)) then {(_this select 2) call (_this select 1)};`
is still faster in my tests than saving them in a variable first and this is a performance critical PFH.